### PR TITLE
fluct RTB spec for a supplier partner

### PR DESCRIPTION
**Pull request by tomill**
Initial version of public doc for publisher partner.

preview link: https://github.com/voyagegroup/fluct-rtb-spec/blob/for-suppliers/for-suppliers.md

review point

- based on an internal discussion in https://docs.google.com/document/d/1-uWNgTWC103LbLnuXJ7ld2Qa2A9dfCYq6Ni8Y63PIPg/edit#
- please check "Fluct RTB Questionnaire Form"  linked
- is the path /for-suppliers.md clear?

TODO

- [x] add examples

### CODE REVIEW

**Comment by ajiyoshi-vg** [comment]
* [x] please check "Fluct RTB Questionnaire Form" linked
  * linked
* [x] is the path /for-suppliers.md clear?
  * maybe

**Review by ajiyoshi-vg — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # fluct Specification
 
-## RTB Spec v2.x
+## RTB Spec for a buyer partner v2.x
```
Comment (yowcow) [inline README.md:3]: :+1:

**Review by yowcow — APPROVED** [review]
Nice! 🥇

**Pull request merged**